appengine/geddy/package.json

```diff
@@ -8,8 +8,9 @@
     "node": "~0.12.7"
   },
   "scripts": {
-    "start": "node node_modules/geddy/bin/cli.js",
-    "debug": "node node_modules/geddy/bin/cli.js --debug",
+    "prestart": "GEDDY_SECRET=config/secrets.json; [[ -f $GEDDY_SECRET ]] || echo '{}' > $GEDDY_SECRET && node node_modules/.bin/geddy gen secret",
```

Comment:
The app won't start without a secret :( A user would have to do this anyway to get the app to run on production, and with this script they don't have to think about it.